Sources/GRPCCore/Documentation.docc/Tutorials/Hello-World/Hello-World.tutorial

```diff
@@ -28,17 +28,23 @@
   @Section(title: "Run a gRPC application") {
     Let's start by running the existing Greeter application.
 
+    As a prerequisite you must have the Protocol Buffers compiler (`protoc`) installed. You can
+    find the instructions for doing this in the [gRPC Swift Protobuf 
+    documentation](https://swiftpackageindex.com/grpc/grpc-swift-protobuf/documentation/grpcprotobuf/installing-protoc).
+    The remainder of this tutorial assumes you installed `protoc` and it's available in
+    your `$PATH`.
+
     @Steps {
       @Step {
-        In a terminal run `swift run hello-world serve` to start the server. By default it'll start
-        listening on port 31415.
+        In a terminal run `PROTOC_PATH=$(which protoc) swift run hello-world serve` to start the
```

Comment:
Should we explain why `PROTOC_PATH` is needed and/or link to the plugin docs that explain it in more detail?

Reply:
Yeah that's a great point, I'll do that.